app/components/shared/persoon/create-persoon.js

```diff
@@ -12,9 +12,8 @@ import {
   isValidRijksregisternummer,
 } from 'frontend-lmb/utils/rijksregisternummer';
 import moment from 'moment';
+import { FEMALE_ID, MALE_ID } from 'frontend-lmb/utils/constants';
```

Comment:
thinking: well-known-ids file? that would keep constants a little more organized too